src/test/java/com/box/sdk/retention/RetentionPolicyTest.java

```diff
@@ -141,5 +179,20 @@ public void createsDefiniteRetentionPolicySuccessfully() throws Exception {
 		assertThat(response.getRetentionLength(), is("30"));
 	}
 
+	@Test
+	@Category(IntegrationTest.class)
+	public void createIndefiniteRetentionPolicyAndRetreiveIndependently() throws Exception {
```

Comment:
Maybe we could merge this into the previous test? Create + Get?

More of a philosophical discussion than anything. I hardly think the test run time is a big factor to worry about optimizing it, so I don't plan on doing much about it!